forcedotcom/commerce-on-lightning
# @W-15803283@ -  BREAKING CHANGE: Migrating from sfdx to sf

### What does this PR do?
Migrating from the SFDX CLI to SF

### What issues does this PR fix or reference?
#<Insert GitHub Issue>, [W-15803283](https://gus.lightning.force.com/a07EE00001rwFaeYAE)

### Functionality Before
Was able to create a scratch org and sample store on the org created.

### Functionality After
we can create a scratch org and sample store on the org created.

### How to Test/Testing Effort 
Verified the scratch org creation locally after updating the commands
<img width="1042" alt="Screenshot 2024-05-27 at 3 22 40 PM" src="https://github.com/forcedotcom/commerce-on-lightning/assets/168570676/d6c549a4-7a58-4fcb-9b2b-c96fbe202c1a">
For store creation, there is an existing issue with the schema where we were unable to deploy the retrieved digital experience bundles due to a schema issue. Will verify this flow once the issue is fixed.
Verified the e2e changes by testing with 250(code line) 
<img width="1366" alt="Screenshot 2024-06-04 at 1 25 16 AM" src="https://github.com/forcedotcom/commerce-on-lightning/assets/168570676/cab999cc-cb92-4fd9-93e0-6cae98903fd9">
